README.md

```diff
@@ -1,7 +1,7 @@
-# Automatic Aggie Scheduler
+# Rev Registration
 
-![Backend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Backend%20CI/badge.svg?branch=backend%2Fmaster)
-![Frontend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Frontend%20CI/badge.svg?branch=frontend%2Fmaster)
+![Backend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Backend%20CI/badge.svg)
+![Frontend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Frontend%20CI/badge.svg)
```

Comment:
```suggestion
[![Backend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Backend%20CI/badge.svg)](https://github.com/aggie-coding-club/rev-registration/actions?query=branch%3Amaster+workflow%3A%22Backend+CI%22)
[![Backend CI](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/workflows/Frontend%20CI/badge.svg)](https://github.com/aggie-coding-club/rev-registration/actions?query=branch%3Amaster+workflow%3A%22Frontend+CI%22)
```
It isn't possible to link directly to the latest run (see [this thread on the github forums](https://github.community/t5/GitHub-Actions/Link-to-the-last-log-from-a-GitHub-Action-test/td-p/43845)), but we can link to a query for the most recent runs on `master`.

Reply:
Yeah I'm cool with that I'll do it that way